km/km_musl_related.c

```diff
@@ -68,8 +71,13 @@ int km_link_map_walk(link_map_visit_function_t* callme, void* visitargp)
    km_gva_t linkmapheadp_gva;
    int rc = 0;
 
-#define KM_DLOPEN_OFFSET_TO_LOAD_HEAD_INSTR 0x11
-#define KM_DLOPEN_OFFSET_TO_POST_LOAD_HEAD_INSTR 0x18
+   static const uint64_t KM_DLOPEN_OFFSET_TO_LOAD_HEAD_INSTR = 17;
+   static const uint64_t KM_DLOPEN_OFFSET_TO_LOAD_HEAD_INSTR_gcc_12 = 20;
+   static const uint64_t KM_DLOPEN_OFFSET_TO_LOAD_HEAD_INSTR_O0 = 45;
```

Comment:
Can you briefly describe the situation where this would happen.
Is optimization disabled?